tests/test_union_city.py

```diff
@@ -105,3 +105,46 @@ def test_highway():
     assert (
         count_different_lines == 0
     ), f"HWYSKIM_MAZMAZ_DA.csv differs on {count_different_lines} lines"
+
+
+@pytest.mark.xfail
+def test_validate_input_fail():
```

Comment:
I'd prefer in general that instead of using `@pytest.mark.xfail` that we catch the exception that we think we will trigger – and then it will fail on anything that we aren't expecting.  Minor in this context, but for the future at least :-)

Reply:
fixed.